src/main/java/Duke.java

```diff
@@ -1,10 +1,109 @@
+import java.util.Scanner;
+
 public class Duke {
+    private static final String LONG_LINE = "____________________________________________________________";
+    private static final String GREETING = "Hello! I'm Duke";
+    private static final String REQUEST = "What can I do for you?";
+    private static final String EXIT = "Bye. Hope to see you again soon!";
+    private static final String ADDED_TASK = "Got it. I've added this task:";
+    private static int totalTaskNumber = 0;
+    private static Task[] tasks = new Task[100];
+
+    public static void printGreeting() {
+        System.out.println(LONG_LINE + System.lineSeparator() + GREETING);
+        System.out.println(REQUEST + System.lineSeparator() + LONG_LINE);
+    }
+
+    public static void addTask(String input) {
+        int slashIndex = input.indexOf('/');
+        Task currentTask;
+        if (input.startsWith("deadline ")) {
+            currentTask = createDeadlineTask(input, slashIndex);
+        }
+        else if (input.startsWith("event ")) {
+            currentTask = createEventTask(input, slashIndex);
+        }
+        else if (input.startsWith("todo ")){
+            currentTask = createTodoTask(input);
+        }
+        else {
+            //Treat all other tasks as Todo tasks
+            currentTask = createDefaultTask(input);
+        }
```

Comment:
I think your if-else statement here should follow the required Java coding standard. 
https://se-education.org/guides/conventions/java/basic.html